Iterable/iterable-web-sdk
# [MOB-7613][MOB-7390] Fix iframe height setter and update vulnerable dependencies

## JIRA Ticket(s) if any

* [MOB-7613](https://iterable.atlassian.net/browse/MOB-7613)
** Fix iframe height setter
* [MOB-7390](https://iterable.atlassian.net/browse/MOB-7390)
** Update vulnerable dependencies

## Test Steps

Given I send myself templates made with BEE and WYSIWYG editors
When I render the messages on my browser
Then I see no unexpected margin around the message
And I see the iframe height is automatically adjusted based on all images loaded in the message

Regression testing on general functionality due to updated dependencies across the board, but esp. for...
display position (center, top right, bottom right, full)
display option (immediate vs deferred)
close button position (top right, top left)
link handling (open new tab, open same tab, external new tab)
...because typings have been refactored for these.

[MOB-7613]: https://iterable.atlassian.net/browse/MOB-7613?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[MOB-7390]: https://iterable.atlassian.net/browse/MOB-7390?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
